packages/botonic-react/src/components/message.jsx

```diff
@@ -313,7 +313,9 @@ export const Message = props => {
               ) : (
                 <BlobText blob={blob}>{textChildren}</BlobText>
               )}
-              {buttons}
+              {!!buttons.length && (
```

Comment:
buttons will always be an array, I think it is not necessary to negate twice (!!)

Reply:
Yes, buttons will always be an array but we need to convert the length value to boolean because otherwise the length value will be displayed in the component